lib/runner.js

```diff
@@ -146,7 +146,9 @@ function runner(script, payload, options) {
   //
   let runnerPlugins = [];
   _.each(runnableScript.config.plugins, function tryToLoadPlugin(pluginConfig, pluginName) {
-    let requireString = 'artillery-plugin-' + pluginName;
+    let pluginPackage = pluginConfig.package || runnableScript.config.pluginsPackage;
+    let pluginPrefix = pluginPackage ? pluginPackage + '/' : 'artillery-plugin-';
```

Comment:
@hassy please look at comment 3 & 4 in the issue below.
https://github.com/shoreditch-ops/artillery-core/issues/108 

Do you suggest we go with my initial suggestion of just reading the "full package name" as provided? Because I've named the config attributes "package" as per your suggestion in comment 4.